packages/intl-durationformat/BUILD

```diff
@@ -1,21 +1,40 @@
 load("@aspect_bazel_lib//lib:write_source_files.bzl", "write_source_files")
+load("@aspect_rules_esbuild//esbuild:defs.bzl", "esbuild")
 load("@aspect_rules_js//npm/private:npm_package.bzl", "npm_package")
 load("@aspect_rules_ts//ts:defs.bzl", "ts_project")
 load("@npm//:defs.bzl", "npm_link_all_packages")
-load("//tools:index.bzl", "check_format", "ts_compile")
+load("@npm//:test262-harness/package_json.bzl", test262_harness_bin = "bin")
+load("//tools:index.bzl", "bundle_karma_tests", "check_format", "generate_src_file", "package_json_test", "ts_compile", "ts_script")
 load("//tools:jest.bzl", "jest_test")
 
+
+
 npm_link_all_packages(name = "node_modules")
 
 PACKAGE_NAME = "intl-durationformat"
 
+
+TEST_LOCALES = [
```

Comment:
Just copied from ListFormat, I haven't put time into building actually useful unit tests for DurationFormat yet - have been focused on test262